src/client/puppet-service.ts

```diff
@@ -564,7 +564,7 @@ export class PuppetService extends Puppet {
   }
 
   private stopGrpcStream (): void {
-    log.verbose('PuppetService', 'stopGrpcStream()')
+    log.info('PuppetService', 'stopGrpcStream()')
```

Comment:
It would be better to keep this as verbose. If we change this to info, developer uses Wechaty will see this in their project and they don't know what to do with it. For debug purpose, we should use verbose level.

Reply:
OK this one is fine.